smtp/smtpc.h

```diff
@@ -267,4 +267,16 @@ smtpc_execute(struct smtpc_request *req, double timeout);
 
 /** Request }}} */
 
+/* {{{ Version */
+
+/**
+ * Get a Curl library version.
+ *
+ * @see https://curl.se/libcurl/c/curl_version_info.html
+ */
+void
+get_curl_version(char *version, size_t len);
```

Comment:
The function doesn't follow the `smtpc_*` name pattern.

Reply:
`smtp*c*_`

It is in the smptc compilation unit. If it shouldn't be there, it is better to move it outside.

However we call libcurl function via this compilation unit, so it seems logical to have it there and have the `smtpc_` prefix.